Source/PINDiskCache.m

```diff
@@ -859,17 +895,22 @@ - (id)objectForKeyedSubscript:(NSString *)key
     NSDate *now = [[NSDate alloc] init];
```

Comment:
Unrelated to this diff, but I just noticed we can save some work by moving this date creation down.